app/src/main/java/openfoodfacts/github/scrachx/openfood/models/Additives.java

```diff
@@ -0,0 +1,29 @@
+package openfoodfacts.github.scrachx.openfood.models;
```

Comment:
@PrajwalM2212 {read the below comment first} as the `AdditiveNmaeDao` returns `AdditiveName` there is no need for this class, it is just redundant.